### PR TITLE
Create sidebar and navbar

### PR DESCRIPTION
**Pull request by KacpiDrewniak**

### CODE REVIEW

File: src/action/action.signIn.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { signInWithEmailAndPassword as signIn, UserCredential } from 'firebase/auth';
+
+import { auth } from '../config';
+
+// eslint-disable-next-line max-len
+export const actionSignIn = (email:string, password:string):Promise<UserCredential> => signIn(auth, email, password);
```
[the remark this answers is not in the document]
Comment (btalar) [reply src/action/action.signIn.ts:6]: try export const actionSignIn = (email:string, password:string):Promise<UserCredential> 
=> signIn(
auth, email, password
);

eslint should formating

File: src/action/action.signOut.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { signOut } from 'firebase/auth';
+
```
Comment (btalar) [inline src/action/action.signOut.ts:2]: Please remove empty Line

File: src/assets/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import logo from './logo.svg';
+
```
Comment (btalar) [inline src/assets/index.ts:2]: rename logo to  signature  -  lowercase

File: src/components/Layout/Layout.styled.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const LayoutStyled = styled.div`
```
Comment (btalar) [inline src/components/Layout/Layout.styled.ts:3]: rename LayoutStyled. to LayoutWrapper

File: src/components/Layout/Layout.styled.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const LayoutStyled = styled.div` 
+  min-height: 100vh;
```
Comment (btalar) [inline src/components/Layout/Layout.styled.ts:4]: IMo here should be height not min-height

File: src/components/Layout/Layout.tsx [hunk 6]
```diff
@@ -1,8 +1,14 @@
 import React, { FC, ReactNode } from 'react';
 
+import { Sidebar } from '../Sidebar';
+import { Container, LayoutStyled } from './Layout.styled';
```
Comment (btalar) [inline src/components/Layout/Layout.tsx:4]: Rename Container to PageWrapper  and please move PageWrapper to separate component add  style should  be 
 centerted and max-width 1200px;

File: src/components/Navbar/Navbar.styled.ts [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import { NavbarMenuToggle } from '@nextui-org/react';
+import styled from 'styled-components';
+
+export const NavbarMenuToggleStyled = styled(NavbarMenuToggle)`
```
Comment (btalar) [inline src/components/Navbar/Navbar.styled.ts:4]: Please not use Styled naming Element   NavbarMenuToggleStyled use  NavbarMenuToggle

File: src/action/action.signIn.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (KacpiDrewniak) [reply src/action/action.signIn.ts:6]: add #TODO

File: src/action/action.signOut.ts (hunk 2 above)
Comment (KacpiDrewniak) [reply src/action/action.signOut.ts:2]: i cannot because eslint formats that

File: src/assets/index.ts (hunk 3 above)
Comment (KacpiDrewniak) [reply src/assets/index.ts:2]: changed to signature

File: src/components/Layout/Layout.styled.ts (hunk 4 above)
Comment (KacpiDrewniak) [reply src/components/Layout/Layout.styled.ts:3]: changed

File: src/components/Layout/Layout.styled.ts (hunk 5 above)
Comment (KacpiDrewniak) [reply src/components/Layout/Layout.styled.ts:4]: changed to hight

File: src/components/Layout/Layout.tsx (hunk 6 above)
Comment (KacpiDrewniak) [reply src/components/Layout/Layout.tsx:4]: rename and set max-width 1200

File: src/components/Navbar/Navbar.styled.ts (hunk 7 above)
Comment (KacpiDrewniak) [reply src/components/Navbar/Navbar.styled.ts:4]: avoid Styled..

**Pull request merged**